.gitattributes

```diff
@@ -1,12 +1,14 @@
-.editorconfig          export-ignore
-.gitattributes         export-ignore
 /.github/              export-ignore
-.gitignore             export-ignore
-.php_cs.dist           export-ignore
 /build/                export-ignore
 /docs/                 export-ignore
+/tests/                export-ignore
+/.editorconfig         export-ignore
+/.gitattributes        export-ignore
+/.gitignore            export-ignore
+/.php_cs.dist          export-ignore
+/Dockerfile            export-ignore
 /Makefile              export-ignore
-/phpstan-baseline.neon export-ignore
 /phpstan.neon.dist     export-ignore
+/phpstan-baseline.neon export-ignore
 /phpunit.xml.dist      export-ignore
-/tests/                export-ignore
+/UPGRADING.md          export-ignore
```

Comment:
IMHO I believe that upgrading guide is something similar to changelog and it should stay